app/controllers/api/scores_controller.rb

```diff
@@ -5,7 +5,19 @@ class ScoresController < ApplicationController
     before_action :validate_score_user_id, only: :destroy
 
     def user_feed
-      scores = Score.all.order(played_at: :desc, id: :desc)
+      scores = Score.all.order(played_at: :desc, id: :desc).includes(:user)
+      serialized_scores = scores.map(&:serialize)
+
+      response = {
+        scores: serialized_scores,
+      }
+
+      render json: response.to_json
+    end
+
+    def user_score
+      user = User.where(name: params[:username]).first
```

Comment:
Instead of doing `.where().first`, use [.find_by()](https://guides.rubyonrails.org/active_record_querying.html#find-by)